spec/features/item_registration_csv_spec.rb

```diff
@@ -82,4 +82,28 @@
       expect(page).to have_content 'Csv file missing headers: source_id.'
     end
   end
+
+  context 'invalid CSV' do
+    it 'reports error' do
```

Comment:
Not a blocker, but this could have been a request test instead. Request tests are much faster and are capable of testing the same behavior that you are testing here.

Reply:
Spent a little time on this, but the complexity of getting the post params correct for this form wasn't worth it.